src/test/miniminer_tests.cpp

```diff
@@ -539,4 +642,64 @@ BOOST_FIXTURE_TEST_CASE(calculate_cluster, TestChain100Setup)
     }
 }
 
+BOOST_FIXTURE_TEST_CASE(manual_ctor, TestChain100Setup)
+{
+    CTxMemPool& pool = *Assert(m_node.mempool);
+    LOCK2(cs_main, pool.cs);
+    {
+        // 3 pairs of fee-bumping grandparent + parent, plus 1 low-feerate child.
```

Comment:
Isn’t it the parent that is fee-bumping?

```suggestion
        // 3 pairs of grandparent + fee-bumping parent, plus 1 low-feerate child.
```